ansible/configs/kni-osp/files/ocp-install.sh.j2

```diff
@@ -260,9 +272,17 @@ prepare_node() {
     *nightly*) export RELEASE_IMAGE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/$VERSION/release.txt| grep 'Pull From: quay.io' | awk -F ' ' '{print $3}' | xargs) ;;
     *) export RELEASE_IMAGE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$VERSION/release.txt| grep 'Pull From: quay.io' | awk -F ' ' '{print $3}' | xargs) ;;
```

Comment:
Is there another way to set RELEASE_IMAGE ?
Is this reliable ? Just wondering if release.txt always present and formatting is consistent everytime

Reply:
Yeah, this is very consistent, and we have a number of documented examples of where people are scraping the release.txt file for the right info.